client/src/App.js

```diff
@@ -1,10 +1,23 @@
-import React from 'react'
-const App = () => {
+
+import React, { Component } from 'react';
+import SearchRectangle from './components/search_info_components/search';
```

Comment:
Can you change SearchRectangle to something more meaningful? SearchBar or something similar would be better